WordPress/twentytwentytwo
# Add About me patterns

**Description**
This adds the patterns from https://github.com/WordPress/twentytwentytwo/issues/15

**Screenshots**
![localhost_4759__p=559](https://user-images.githubusercontent.com/275961/137289582-d8886d60-3d4a-4fbe-9b5b-5091a8b8cd53.png)

**Testing Instructions** 
1. Add each of the patterns from the block inserter
2. You will need to set a site logo and have some pages to see all aspects of the patterns
